linera-service/src/linera/client_context.rs

```diff
@@ -195,7 +147,7 @@ impl ClientContext {
             #[cfg(feature = "rocksdb")]
             None => {
                 let storage_config = linera_service::storage::StorageConfig::RocksDb {
-                    path: Self::create_default_config_path()?.join("wallet.db"),
```

Comment:
(Was that a bug?)

Reply:
I don't think so — there's currently no way to change the config path, so the code is equivalent.  But we have such a way for the wallet path, so I thought I might as well move this one so we're ready when we inevitably make it configurable :)